pkg/operator/operator.go

```diff
@@ -274,15 +275,21 @@ func refToFileScheme(ref imgreference.DockerImageReference) string {
 // Copied from https://github.com/openshift/oc/blob/4df50be4d929ce036c4f07893c07a1782eadbbba/pkg/cli/admin/catalog/mirror.go#L284-L313
 // Hoping this can be temporary, and `oc adm mirror catalog` libs support index.yaml direct mirroring.
 
-func newMirrorerFunc(opts *catalog.MirrorCatalogOptions) (catalog.ImageMirrorerFunc, error) {
+func newMirrorerFunc(cfg v1alpha1.ImageSetConfiguration, opts *catalog.MirrorCatalogOptions) (catalog.ImageMirrorerFunc, error) {
 	allmanifestsFilter := imagemanifest.FilterOptions{FilterByOS: ".*"}
 	if err := allmanifestsFilter.Validate(); err != nil {
 		return nil, err
 	}
 
 	return func(mapping map[imagesource.TypedImageReference]imagesource.TypedImageReference) error {
+
 		mappings := []imgmirror.Mapping{}
 		for from, to := range mapping {
+
+			if bundle.IsBlocked(cfg, from.Ref) {
+				logrus.Warnf("image %s was specified as blocked per config, skipping...", from.Ref.Name)
```

Comment:
Since this is a user-configured action, I'd just debug log this.
```suggestion
				logrus.Debugf("image %s was specified as blocked per config, skipping...", from.Ref.Name)
```